jdaviz/core/tools.py

```diff
@@ -249,3 +249,44 @@ class SidebarShortcutCompass(_BaseSidebarShortcut):
     tool_id = 'jdaviz:sidebar_compass'
     action_text = 'Compass'
     tool_tip = 'Open compass plugin in sidebar'
+
+
+@viewer_tool
+class SinglePixelRegion(CheckableTool):
+
+    icon = os.path.join(ICON_DIR, 'select_single_pixel.svg')
+    tool_id = 'jdaviz:singlepixelregion'
+    action_text = 'Create single-pixel spatial region'
+    tool_tip = 'Define a single-pixel spatial region of interest'
+
+    def activate(self):
+        self.viewer.add_event_callback(self.on_mouse_event, events=['click'])
+
+    def deactivate(self):
+        self.viewer.remove_event_callback(self.on_mouse_event)
```

Comment:
When combined with #1679, this will need to be updated in order to ensure that the subset selection mode reverts to "Create New", as expected.  I think this should still be safe even before 1679, but please test first.

```suggestion
class SinglePixelRegion(BqplotSelectionTool):

    icon = os.path.join(ICON_DIR, 'select_single_pixel.svg')
    tool_id = 'jdaviz:singlepixelregion'
    action_text = 'Create single-pixel spatial region'
    tool_tip = 'Define a single-pixel spatial region of interest'

    def activate(self):
        super().activate()
        self.viewer.add_event_callback(self.on_mouse_event, events=['click'])

    def deactivate(self):
        self.viewer.remove_event_callback(self.on_mouse_event)
        super().deactivate()
```

Reply:
I'll accept the suggestion first and go from there. Thanks!